doc/examples/example.sh

```diff
@@ -13,7 +13,7 @@ g++ -fprofile-arcs -ftest-coverage -fPIC -O0 example.cpp -o program
 #END run
 
 #BEGIN gcovr
-gcovr
+gcovr -o example.txt
 #END gcovr
```

Comment:
The  `-o` should of course be changed to `--html` but the file needs to be given or the documentation text needs to be updated because the output file name in mentioned:
https://github.com/gcovr/gcovr/blob/3e121a212d1219684b28165701af2a70112d0bbf/doc/source/output/html.rst?plain=1#L14-L16